library/alloc/src/vec.rs

```diff
@@ -194,6 +194,19 @@ use crate::raw_vec::RawVec;
 /// can be slow. For this reason, it is recommended to use [`Vec::with_capacity`]
 /// whenever possible to specify how big the vector is expected to get.
 ///
+/// In summary, a vector containing 1, 2 with capacity 4 can be visualized as:
+/// ```text
+/// Stack  +--------+--------+--------+
+///        |    ptr |capacity|    len |
```

Comment:
Could you center "ptr" and "len", please? Same for all the values. :)

Reply:
Good idea. By the way, I purposely make it 8 spaces.

```suggestion
///        |   ptr  |capacity|   len  |
```